src/interpreter/ir_normalisation.jl

```diff
@@ -56,6 +56,7 @@ function _interpolate_boundschecks!(statements::Vector{Any})
         if stmt isa Expr && stmt.head == :boundscheck && length(stmt.args) == 1
             def = SSAValue(n)
             val = only(stmt.args)
+            # TODO: this could just be `statements[n] = val` (Valentin C says)
```

Comment:
@vchuravy would you be able to expand on this at all? I'm not sure what is meant here.

Reply:
I think this is exactly #397?

Reply:
Ahhh okay, great. Thanks for clarifying.